python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -582,9 +582,6 @@ def makePipeline(self, args: SimpleNamespace) -> Pipeline:
         if args.save_pipeline:
             pipeline.write_to_uri(args.save_pipeline)
 
-        if args.pipeline_dot:
-            pipeline2dot(pipeline, args.pipeline_dot)
```

Comment:
I am now thinking how this PR relates to #159 given that this is the only usage of pipeline2dot and that PR was removing it completely.

Reply:
I think this fully supersedes #159.  I'm also planning to RFC dropping support for QuantumGraph GraphViz outputs, since my understanding is that those are always too big for `dot` to render in practice anyway, and then we can get rid of `dotTools.py` entirely.